Documentation/MigrateToComposer/MigrationSteps.rst

```diff
@@ -68,30 +68,22 @@ In short:
 
 You have to decide by yourself, which syntax fits best to your needs.
 
-Install the Core
-----------------
-
 .. _composer-migration-require-all:
+.. _composer-migration-require-subtree-packages:
 
-The Old Way: Add Everything
-~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-As already written above, the line to install TYPO3 7 LTS would be::
-
-    composer require typo3/cms:~7.6.0
+Install the Core
+----------------
 
-.. _composer-migration-require-subtree-packages:
+.. hint::
 
-Add only code you need
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+   Since TYPO3 9, TYPO3 must be installed as individual packages for
+   each system extension (see subtree split).
+   The concept means, you will not copy
+   the full TYPO3 core package, including all system extensions, you will
+   never use. But only install what you really want. You will not be able
+   to install the full core `typo3/cms:^9`.
```

Comment:
Suggestion:

> Since version 9 TYPO3 must be installed using individual `typo3/cms-*` packages (see "subtree split" for details). This means that you will only install the system extensions you really need which among others increases security. The former `typo3/cms` package cannot be installed anymore.

Reply:
Very nice. I used your suggested text.